app/konnect/runtime-manager/composite-runtime-groups/how-to.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Working with composite runtime groups
+content_type: how-to
+badge: enterprise
+---
+
+A composite runtime group (CRG) is a type of runtime group in {{site.konnect_short_name}} where `cluster_type = CLUSTER_TYPE_COMPOSITE`.
+
+In this guide, you will set up a composite runtime groups with two members, then test that configuration from both member groups is applied to the composite.
+
+## Prerequisites
+
+* You must have the **runtime group admin** role to fully manage composite runtime groups.
+* If you are using the {{site.konnect_short_name}} API, you have a personal or system access token.
+
+## Using composite runtime groups
+
+### Set up standard runtime groups
+
+First, let's create a standard runtime group. 
+This group will be a member of a composite runtime group later on.
+
+If you already have some standard runtime groups in your org that you want to add to a composite, skip to [creating a composite runtime group](#create-composite-runtime-group).
+
+{% navtabs %}
+{% navtab Konnect UI %}
+
+1. From the navigation menu, open {% konnect_icon runtimes %} **Runtime Manager**.
+1. Click the **New Runtime Group** button and select **{{site.base_gateway}}**.
+   
+   Kong Ingress Controller runtime groups can't be part of composite groups.
```

Comment:
We should also add that composite RGs cannot be a member of another composite RG